integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -726,12 +724,10 @@ fn test_resharding_v3_resharding_block_in_fork() {
 }
 
 #[test]
-// TODO(resharding): fix nearcore and un-ignore this test
 // TODO(resharding): duplicate this test so that in one case resharding is performed on block
 //                   B(height=13) and in another case resharding is performed on block B'(height=13).
 //                   In the current scenario the real resharding happens on block B'. Low priority TODO
 //                   since it's a very rare corner case.
```

Comment:
Is this todo still to do?

Reply:
Yes this must be done still. `split_shard_in_forks_when_older_branch_is_finalized` is doing that but only for flat storage unit testing; it would be nice to have the test loop version with all the components.